suite/smoke/ocr_test.go

```diff
@@ -2,167 +2,77 @@ package smoke
 
 import (
 	"context"
-	"fmt"
-	"math/big"
-	"time"
-
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
-	uuid "github.com/satori/go.uuid"
 	"github.com/smartcontractkit/helmenv/environment"
 	"github.com/smartcontractkit/helmenv/tools"
 	"github.com/smartcontractkit/integrations-framework/actions"
-	"github.com/smartcontractkit/integrations-framework/client"
-	"github.com/smartcontractkit/integrations-framework/contracts"
 )
 
 var _ = Describe("OCR Feed @ocr", func() {
 	var (
-		err             error
-		nets            *client.Networks
-		cd              contracts.ContractDeployer
-		lt              contracts.LinkToken
-		ocr             contracts.OffchainAggregator
-		ocrRoundTimeout = 2 * time.Minute
-		cls             []client.Chainlink
-		adapterPath     string
-		mockserver      *client.MockserverClient
-		e               *environment.Environment
+		err      error
+		env      *environment.Environment
+		ocrSetup *actions.OCRSetup
 	)
 	BeforeEach(func() {
 		By("Deploying the environment", func() {
-			e, err = environment.DeployOrLoadEnvironment(
+			env, err = environment.DeployOrLoadEnvironment(
 				environment.NewChainlinkConfig(environment.ChainlinkReplicas(6, nil)),
 				tools.ChartsRoot,
 			)
 			Expect(err).ShouldNot(HaveOccurred())
-			err = e.ConnectAll()
+			err = env.ConnectAll()
 			Expect(err).ShouldNot(HaveOccurred())
 		})
-
 		By("Getting the clients", func() {
```

Comment:
I prefer the pattern:
```go
By("Getting the clients", actions.NewOCRSetup(env))
```
You could keep the functions in the actions the same, but wrap them to handle error assertions, eg:

```go
func GinkgoWrap(fn func() error) func() {
  return func() {
    Expect(fn()).ShouldNot(HaveOccurred())
  }
}
```